elex/api/models.py

```diff
@@ -876,6 +876,7 @@ def __init__(self, **kwargs):
         self.setzerocounts = kwargs.get('setzerocounts', False)
 
         self.raceids = kwargs.get('raceids', [])
+        self.officeids = kwargs.get('officeids', "")
```

Comment:
Add a comment here to explain why `officeids` are being handled as a string instead of a list of strings; this may not be clear to future devs.

Reply:
Also, should the default value be `""`, or `None`? I genuinely don't know; you have a better grasp on how `requests` will submit this querystring.

Reply:
On second comment - After poking around a little more, I don't think it matters at this particular point because this particular line is executed before the command line args are parsed. But I do think `None` is a cleaner default value!